.github/workflows/performance-integration-tests.yml

```diff
@@ -8,9 +8,8 @@ on:
   #   - https://crontab.guru/
   schedule:
     # Runs every 1 hours.
```

Comment:
NIT: Also change this to `every 4 hours` and remove the TODO below.

Reply:
Done